Server/JokesOnYou.Web.Api/Repositories/JokesRepository.cs

```diff
@@ -37,5 +40,19 @@ public JokesRepository(DataContext context, IMapper mapper)
             await _context.Jokes.ProjectTo<JokeReplyDto>(_mapper.ConfigurationProvider)
                                 .FirstOrDefaultAsync(j => j.Id == id);
         public void DeleteJoke(Joke joke) => _context.Jokes.Remove(joke);
+
+        public async Task<IEnumerable<JokeReplyDto>> GetFilterdJokesAsync(IEnumerable<IFilter> filters)
+        {
+            var jokes = from j in _context.Jokes
```

Comment:
We are not using query syntax in other places so might be a good idea to be consistent